packages/tools/client-debugger/client-debugger-view/src/Utilities.ts

```diff
@@ -16,14 +16,7 @@ import { IResolvedUrl } from "@fluidframework/driver-definitions";
  * @internal
  */
 export function resolvedUrlToString(resolvedUrl: IResolvedUrl): string {
```

Comment:
This function is kind of useless now.  Suggest inlining `resolvedUrl.url` in usage locations, if any exist and delete function.